pkgs/gnolang/store.go

```diff
@@ -370,7 +371,7 @@ func (ds *defaultStore) DelObject(oo Object) {
 func (ds *defaultStore) GetType(tid TypeID) Type {
 	tt := ds.GetTypeSafe(tid)
 	if tt == nil {
-		ds.Print()
+		// ds.Print()
```

Comment:
Leftover?

Reply:
It's probably better if we don't have this, this was what was generating the thousands of lines of useless logs before the actual error